File: onekk/src/main/java/org/wheatgenetics/onekk/IntroActivity.java
package org.wheatgenetics.onekk;

import android.graphics.Color;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.os.Bundle;
import com.github.paolorotolo.appintro.AppIntro2;
import com.github.paolorotolo.appintro.AppIntroFragment;

public class IntroActivity extends AppIntro2 {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        // Customize view
        showStatusBar(false);

        // Add slides
        addSlide(AppIntroFragment.newInstance(getString(R.string.intro_title_1), getString(R.string.intro_body_1), R.drawable.ic_help, Color.parseColor("#A84937")));
        addSlide(AppIntroFragment.newInstance(getString(R.string.intro_title_2), getString(R.string.intro_body_2), R.drawable.ic_help, Color.parseColor("#285E3D")));
        addSlide(AppIntroFragment.newInstance(getString(R.string.intro_title_3), getString(R.string.intro_body_3), R.drawable.ic_help, Color.parseColor("#0C6291")));

        // Hide Skip/Done button.
        showSkipButton(false);
        setProgressButtonEnabled(true);
    }

    @Override
    public void onSkipPressed(Fragment currentFragment) {
        super.onSkipPressed(currentFragment);
    }

    @Override
    public void onDonePressed(Fragment currentFragment) {
        super.onDonePressed(currentFragment);
        finish();
    }

    @Override
    public void onSlideChanged(@Nullable Fragment oldFragment, @Nullable Fragment newFragment) {
        super.onSlideChanged(oldFragment, newFragment);
    }
}

File: onekk/src/main/java/org/wheatgenetics/onekk/activities/IntroActivity.kt
package org.wheatgenetics.onekk.activities

import android.annotation.SuppressLint
import android.graphics.Color
import android.os.Bundle
import androidx.fragment.app.Fragment
import com.github.appintro.AppIntro
import com.github.appintro.AppIntroBase
import com.github.appintro.AppIntroFragment
import org.wheatgenetics.onekk.R

class IntroActivity : AppIntro() {
    @SuppressLint("MissingSuperCall") //AS bug?
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        // Make sure you don't call setContentView!

        // Call addSlide passing your Fragments.
        // You can use AppIntroFragment to use a pre-built fragment
        addSlide(AppIntroFragment.newInstance(
                title = getString(R.string.intro_slide_1), //Welcome to OneKK
                backgroundColor = getResources().getColor(R.color.colorPrimaryDark),
                description = getString(R.string.intro_slide_1_description),
                imageDrawable = R.drawable.ic_launcher
        ))
        addSlide(AppIntroFragment.newInstance(
                title = getString(R.string.intro_slide_2), //OneKK works with a lightbox, four coins, and a Bluetooth Ohaus scale.
                backgroundColor = getResources().getColor(R.color.colorPrimaryDark),
                description = getString(R.string.intro_slide_2_description),
                imageDrawable = R.drawable.scale
        ))
        addSlide(AppIntroFragment.newInstance(
                title = getString(R.string.intro_slide_3),
                backgroundColor = getResources().getColor(R.color.colorPrimaryDark),
                description = getString(R.string.intro_slide_3_description),
                imageDrawable = R.drawable.sample_original
        ))
        addSlide(AppIntroFragment.newInstance(
                title = getString(R.string.intro_slide_4),
                backgroundColor = getResources().getColor(R.color.colorPrimaryDark),
                description = getString(R.string.intro_slide_4_description)
        ))
    }

    override fun onSkipPressed(currentFragment: Fragment?) {
        super.onSkipPressed(currentFragment)
        // Decide what to do when the user clicks on "Skip"
        finish()
    }

    override fun onDonePressed(currentFragment: Fragment?) {
        super.onDonePressed(currentFragment)
        // Decide what to do when the user clicks on "Done"
        finish()
    }
}